### PR TITLE
Refactor to textview

### PR DESCRIPTION
**Pull request by hurrba**
## Status
**READY**

## Description
Refactored text to be a `TextView` to allow for easier customization. 

## Related PRs/Issues
This solves tickets #46 and #51 requesting to [set custom font](https://github.com/cortinico/slidetoact/issues/46), and [text to all caps](https://github.com/cortinico/slidetoact/issues/51).

## Todos
I haven't added screens/text to README, waiting for this to be reviewed first. 
Also versioning is missing. 

## Feedback Reviews

@cortinico

### CODE REVIEW

**Comment by hurrba** [comment]
@cortinico circleci fails to build with: 
> Failed to install the following Android SDK packages as some licences have not been accepted.
>platforms;android-27 Android SDK Platform 27

what is the expected timeline on next release? Can this make the next release?

**Comment by cortinico** [comment]
> what is the expected timeline on next release? Can this make the next release?

Definitely. This should go in the `0.6.0`.
I don't have a strict timeline for releases, but I can prepare one for next week :) In the meanwhile you can still use jitpack.io if you really need to use `master`.

> @cortinico circleci fails to build with:
> Failed to install the following Android SDK packages as some licences have not been accepted.
> platforms;android-27 Android SDK Platform 27

I'll take care of this 👍

File: slidetoact/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -8,6 +8,8 @@
             <enum name="bold" value="1" />
             <enum name="italic" value="2" />
         </attr>
+        <attr name="text_all_caps" format="boolean" />
```
Comment (cortinico) [inline slidetoact/src/main/res/values/attrs.xml]: So I think that maybe having those `text_` attributes was a bad API decision.
I'd rather got for having a:
```suggestion
        <attr name="android:textAppearance" />
```
And allow customization on the style level.
We'll keep on having the other `text_` attributes for backward compatibility.

**Comment by cortinico** [comment]
Hey @hurrba
The CircleCI build failure is now fixed. You should be able to rebase on master

**Comment by slegaitis** [comment]
Hey, any update on this? :)

**Comment by hurrba** [comment]
Sorry, I haven't had time to look at this again, and tbh it's quite far down on my todo list right now. 

I'll try to find some time in the coming days

**Pull request closed without merging**